test/sanitize_json.d

```diff
@@ -39,25 +39,44 @@ int main(string[] args)
     }
 
     auto json = parseJSON(readText(inFilename));
-    sanitize(json.array);
+    sanitize(json);
 
-    outFile.write(json.toJSON(true));
+    outFile.write(json.toJSON(true, JSONOptions.doNotEscapeSlashes));
     return 0;
 }
 
-void sanitize(JSONValue[] rootArray)
+string capitalize(string s)
 {
-    foreach (ref obj; rootArray)
+    if (s.length == 0 || (s[0] < 'a' || s[0] > 'z'))
+        return s;
+    return (cast(char)(s[0] - ('a' - 'A'))) ~ s[1 .. $];
+}
```

Comment:
https://dlang.org/phobos/std_string.html#.capitalize ?

Reply:
Looks like that one won't work, it converts the rest to lower-case...which is not what is wanted.

Reply:
Ah that's unexpected. I thought there's a function for this in Phobos :/
Well, you could do this:

```d
import std. unit : asCapitalized;
return s.take(1).asCapitalized.chain(s.drop(1));
```

It would be more idiomatic, but put more stress on the CTFE (though it's a small standalone script for which this is probably not a concern);

Reply:
Ugh...I can't get it to work.  I added `.array` to the end but it ends up returning `dchar[]`...

Reply:
Interesting it worked for me (https://run.dlang.io/is/xP19Bv), 

>  the end but it ends up returning dchar[]

You can use `.byCodeUnit` to opt-out of auto-decoding, but `to!string` or `text` from `std.conv` are both smart enough (e.g. https://run.dlang.io/is/zv8RlX)

For pure ASCII, there's also `std.ascii.toLower`:

```d
s.enumerate.map!(a => a.index == 0 ? a.value.toUpper : a.value).text;
```

https://run.dlang.io/is/rHnADh

But this is definitely nit-picking here ;-)

Reply:
looks like `std.conv.text` did the trick.  My `std.range` / `std.algorithm` 'fu could use work.